.github/workflows/ldk-node-integration.yml

```diff
@@ -30,20 +29,105 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: vss-server
+
       - name: Checkout LDK Node
         uses: actions/checkout@v3
         with:
           repository: lightningdevkit/ldk-node
           path: ldk-node
 
-      - name: Build and Deploy VSS Server
```

Comment:
While these ~unrelated changes here and elsewhere look fine, I want to note that it would have been preferable if they happened in separate (prefactor) commits. It would at the very least have made reviewing a lot easier.